### ISSUE TITLE
Does not work with Xcode 10.1

### ISSUE THREAD

**Issue by phuongzzz** [XVimProject/XVim2#199]
I've installed Xvim2 successfully with Xcode 10.0 before.
Today I've update to Xcode 10.1 and XVim2 does not work anymore
I've tried to rebuild Xvim2 again, pull the newest code from Xvim2 repository but the problem is still there?
Anyone have a solution? :D

**Comment by 0ber** [comment]
I just installed xvim2 in xcode 10.1 it works, you need remove xcodeSigner and reinstall again https://github.com/XVimProject/XVim2/blob/master/SIGNING_Xcode.md

**Comment by jfoster39** [comment]
Having the same problem with 10.1, but the normal recreation of the signing cert and install process isn't working for me. 

Running `defaults delete com.apple.dt.Xcode DVTPlugInManagerNonApplePlugIns-Xcode-10.1` results in a `Domain (com.apple.dt.Xcode) not found.` message.

It seems like this would be causing the code bundle for xvim2 to fail to load, anybody else having a similar issue?

**Comment by jfoster39** [comment]
> Having the same problem with 10.1, but the normal recreation of the signing cert and install process isn't working for me.
> 
> Running `defaults delete com.apple.dt.Xcode DVTPlugInManagerNonApplePlugIns-Xcode-10.1` results in a `Domain (com.apple.dt.Xcode) not found.` message.
> 
> It seems like this would be causing the code bundle for xvim2 to fail to load, anybody else having a similar issue?

If anybody runs into this, the solve was to completely remove and reclone the repo for xvim2 - not sure why it worked but something was probably ran as sudo at some point causing perm errors down the line.

**Comment by phuongzzz** [comment]
@jfoster39 
Have you solved your problem?
My first attempt is rebuild XVim2, and it does not working with XCode 10.1, and I see that the cloned repo has some changes
So instead of remove the cloned repository, I just reset hard that repo, pull the master branch (but the master branch does not have any update at the time I pull it :D). After that, create new certificate (you can remove the old certificate, but I see it does not really matters) and then resign XCode with that certificate -> rebuild -> and everything works properly

**Comment by wangjiejacques** [comment]
not works in Xcode 10.1

**Comment by daiki1003** [comment]
wmm... it doesn't work even removing the XcodeSigner.

**Comment by daiki1003** [comment]
finally, it works perfectly! 
pull new commit `76852cf78332f53252eadb4f281f6530fbd2cb5f: xcode10.1 uuid` , rebuild XVim2, and relaunch XCode.

**Comment by yingmu52** [comment]
confirmed working in Version 10.1 (10B61)

you need to recode sign and reinstall

**Comment by susan51531** [comment]
Run `make clean && make uninstall`, if you have pulled new commits.

**Comment by josoriora** [comment]
> > Having the same problem with 10.1, but the normal recreation of the signing cert and install process isn't working for me.
> > Running `defaults delete com.apple.dt.Xcode DVTPlugInManagerNonApplePlugIns-Xcode-10.1` results in a `Domain (com.apple.dt.Xcode) not found.` message.
> > It seems like this would be causing the code bundle for xvim2 to fail to load, anybody else having a similar issue?
> 
> If anybody runs into this, the solve was to completely remove and reclone the repo for xvim2 - not sure why it worked but something was probably ran as sudo at some point causing perm errors down the line.

This really did the trick for me. Thank you @jfoster39

**Comment by ekstasis** [comment]
macOS 10.13.6  //  Xcode 10.1 
5c7aaa5 (HEAD -> master, origin/master, origin/develop, origin/HEAD) XcodeHeader directory

I've tried a few combinations for uninstalling, recloning and re-signing, and still XCode is not noticing any change.  FWIW, com.apple.dt.Xcode DVTPlugInManagerNonApplePlugIns-Xcode-10.1  does not exist.

Was it recommended/required to delete the previous certificate and create a new one when re-signing XCode?  I've only tried this once or twice -- maybe not in the right combination of uninstall/reclone/resign.  Or maybe there's another step altogether I haven't seen mentioned?

This is the exact sequence of my most recent try:

[From Issue 146:](https://github.com/XVimProject/XVim2/issues/146#issuecomment-390576227)
	1	Clone https://github.com/XVimProject/XVim2
	2	cd xvim2
	3	make uninstall
	4	Code sign xcode~
	5	Back to xvim2 dir -> make

(of course step 0 is rm -Rf XVim2).  Also notice no certificate deletion.

Any ideas??
